catalog/templates/base_generic.html

```diff
@@ -17,13 +17,32 @@
       {% block sidebar %}
         <ul class="sidebar-nav">
           <li><a href="{% url 'index' %}">{% trans 'Home' %}</a></li>
-          <li><a href="">{% trans 'All books' %}</a></li>
-          <li><a href="">{% trans 'All authors' %}</a></li>
+          <li><a href="{% url 'books' %}">{%trans 'All books'%}</a></li>
+          <li><a href="{% url 'authors' %}">{%trans 'All authors'%}</a></li>
         </ul>
      {% endblock %}
       </div>
       <div class="col-sm-10 ">{% block content %}{% endblock %}</div>
-    </div>
+      <div class="col-sm-10 ">
+      {% block pagination %}
```

Comment:
căn lề vào ra cho đúng indent em hè (vd: indent = 4 spaces)
```suggestion
          {% block pagination %}
              {% if is_paginated %}
```

Reply:
da chi